docs/source/development/config.rst

```diff
@@ -93,7 +90,7 @@ Profiles
 ========
 
 A profile is a directory containing configuration and runtime files, such as
-logs, connection info for the parallel apps, and your IPython command history.
+logs, connection info, and your IPython command history.
```

Comment:
It no longer contains connection info normally, because that's in a Jupyter location. Maybe it does for ipyparallel (@minrk?), but that's a bit confusing, so I'd probably not mention it here.